OvmfPkg/LoongArchVirt/PlatformPei/Platform.h

```diff
@@ -0,0 +1,144 @@
+/** @file
+  Platform PEI module include file.
+
+  Copyright (c) 2024 Loongson Technology Corporation Limited. All rights reserved.<BR>
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef PLATFORM_H_
+#define PLATFORM_H_
+
+#include <IndustryStandard/E820.h>
+#include <IndustryStandard/Pci22.h>
+#include <Library/CpuMmuLib.h>
+#include <Uefi/UefiSpec.h>
+
+typedef struct {
+  UINT64    BaseAddr;
+  UINT64    Length;
+  UINT32    Type;
+  UINT32    Reserved;
+} MEMMAP_ENTRY;
+
+/**
+  Create a page table and initialize the memory management unit(MMU).
+
+  @param[in]   MemoryTable           A pointer to a memory ragion table.
+
+  @retval  EFI_SUCCESS                Configure MMU successfully.
+           EFI_INVALID_PARAMETER      MemoryTable is NULL.
+           EFI_UNSUPPORTED            MemoryRegionMap failed or out of memory space or size not aligned
+                                      or MaxLivel out of bound.
+**/
+EFI_STATUS

```

Comment:
Why is this declared here? Please use a library class declaration/definition

Same for all other forward declarations in this file - only things that are shared between the .c files in this module belong here.